crates/bevy_window/src/window.rs

```diff
@@ -1006,6 +1011,11 @@ pub enum WindowMode {
     /// After that, the window's physical size will be modified to match
     /// that monitor resolution, and the logical size will follow based on the
     /// scale factor, see [`WindowResolution`].
+    ///
+    /// Note: As this mode respects the scale factor provided by the operating system,
+    /// the window's logical size may be different from its physical size.
+    /// If you want to avoid that behavior, you can use the [`WindowResolution::set_scale_factor_override`]
+    /// function to set the scale factor to 1.0.
```

Comment:
Same here:

```suggestion
    /// If you want to avoid that behavior, you can use the [`WindowResolution::set_scale_factor_override`] function
    /// or the [`WindowResolution::with_scale_factor_override`] builder method to set the scale factor to `1.`.
```
```